Harude eelvaated

tormi opened the pull request:
Ma ei tea, kas pealkiri on kõige õnnestunum, aga idee on näidata, millega tegu ;)

tormi committed:
Harude eelvaated

tormi committed:
Harud > Näited

tkardi commented:
:+1: See on väga hea! Ma kohendan natuke seletavat teksti homme-ülehomme. Kas Sulle sobib kui teen commiti otse patchi või eelistad, et teen pull requesti pull requestile? :)

tormi commented:
Otse ikka ;)

tormi commented:
@tkardi , vaata korra develi eelvaate konsooli, seal sellised veateated:

`gsavalik.envir.ee/geoserver/gwc/service/tms/1.0.0/keskkonnainfo:ristipuud@EPSG:3301@geojson/3/3/3.geojson Failed to load resource: the server responded with a status of 404 (Not Found) `

tormi committed:
master > GH Pages

tormi commented:
Muutsin `master` haru aadressi. Selle töölehakkamiseks tuleb seadetes sisse lülitada GitHub Pages ja repoks valida `master` (tee see enne commit'i ära - siis muudatus koodis annab signaali, et GH Pages on vaja uuesti ehitada ja [leht](https://e-gov.github.io/thema/kaart/thema/) hakkab tööle).

tkardi commented:
> the server responded with a status of 404 (Not Found)

See "peabki" olema. Neid kahhleid pole olemas, sest seal pole andmeid.
 
Ja lülisin GH pagesi ka sisse.

tormi commented:
:+1:

tkardi committed:
Update README.md

Näidete tekstiosa tsiba pikemaks kirjutatud. `master`i paragrahvist võiks viide olla äkki paigalduse peatükile (kui see luua s.t) ja `devel`is üleskutse kaastööle?

tkardi commented:
Nii keeruline on kergestimõistetavat teksti kirjutada :(
Kui sinu poolest sobib, mestin tagasi.

tormi commented on README.md:
```diff
@@ -11,6 +11,7 @@ teemakaardirakendused.
-- e-gov/thema [master](https://e-gov.github.io/thema/kaart/thema/) haru
-- e-gov/thema [devel](http://htmlpreview.github.io/?https://github.com/e-gov/thema/blob/devel/kaart/thema/index.html) haru
+## thema teemakaartide näited
+**thema** ülesehitamisel püüame lähtuda sellest, et teemakaardirakendusi oleks võimalik muu hulgas hallata GitHubis. Olemasolevast funktsionaalsusest saab ülevaate e-gov/thema repo [master](https://e-gov.github.io/thema/kaart/thema/) haru *šabloonrakenduse* eelvaatest. 
```
Muudaks veidi:
- _oleks võimalik muu hulgas hallata GitHubis_ > _saaks hallata ka GitHub'is_
- _šabloonrakenduse_ > _mallrakenduse_ või _näidisrakenduse_

tormi committed:
Pisitäiendused

tormi committed:
Veel pisitäiendusi

tkardi commented:
:+1: mestin tagasi
